code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -1207,6 +1207,36 @@
 	if (prob(dose))
 		M.vomit()
 
+
+/datum/reagent/mental/kokoreed
+	name = "KoKo Reed Juice"
+	id = "kokoreed"
+	description = "A tall, green, grass-like plant grows up to 7 feet tall. In the past it thrived in the wet marshlands outside Iz'iz, having evolved to take nutrients from the otherwise toxic byproducts of the algae-contaminated waters. It was discovered by the Unathi colonists that the stalk can be chewed to induce a soothing sense of warmth and comfort, but only to Unathi due to its unique interactions with Unathi physiology. Other species simply taste a sweet, stringy reed. ."
```

Comment:
This description is way too long dude. Its a candy bar reagent for god's sake.

Reply:
Extra lore can be moved to description_fluff